### ISSUE TITLE
Locate Button doesn't locate drone

### ISSUE THREAD

**Issue by vsibanyo** [DJI-Mobile-SDK-Tutorials/Android-GSDemo-GoogleMap#16]
Hi,

I'm having problems with the locate button using the DJI simulator and the GSDemo android app, the drone is connected to my laptop using a USB cable, and my android device is connected to the Spark's wi-fi network. 
Whenever I press the locate button the google maps view just takes me to Egypt or the Atlantic ocean. Has anyone ever experienced this issue?

**Comment by dji-dev** [comment]
Hi @vsibanyo , Spark doesn't support Waypoint Mission, so this demo doesn't work for Spark. So the locate button won't work for Spark.

**Comment by elik21** [comment]
Yes  on locate click it takes me to the sea without any airplane shown
